src/rules/relay_connections_have_fields.js

```diff
@@ -0,0 +1,38 @@
+import { ValidationError } from '../validation_error';
+const MANDATORY_FIELDS = ['pageInfo', 'edges'];
+
+function validateConnection(context, node) {
+  const typeName = node.name.value;
+  if (!typeName.endsWith('Connection')) {
+    return;
+  }
+  const fieldNames = node.fields.map(field => field.name.value);
+  const missingFields = MANDATORY_FIELDS.filter(
```

Comment:
What are your thoughts on also validating the type of `pageInfo` and `edges`?

> A “Connection Type” must contain a field called `edges`. This field must return a list type that wraps an edge type, where the requirements of an edge type are defined in the “Edge Types” section below.

> A “Connection Type” must contain a field called `pageInfo`. This field must return a non‐null  `PageInfo` object, as defined in the “PageInfo” section below.

Reply:
To be done later.™
😄